srcpkgs/Solaar/template

```diff
@@ -17,4 +17,5 @@ checksum=2b292da8923e19a4a7a459d6fcd1119ce157cb579fd00077f87633c0dedfbba5
 
 post_install() {
 	vdoc "${FILESDIR}/README.voidlinux"
+	vinstall "rules.d/42-logitech-unify-permissions.rules" 644 /usr/share/solaar/udev-rules.d
```

Comment:
why do they need to be installed to `/usr/share/solaar`? udev doesn't read that, and they are in `/usr/lib/udev/rules.d` (the proper place) already

Reply:
The included `README.voidlinux` file mentions copying (or symlinking) the file from that location.

Reply:
I believe I see what happened now. An earlier version of the package _did_ need a symlink or copy from `/usr/share/solaar`, and a newer version broke the link for me which caused udev to misbehave. Having removed the link entirely and reloaded the rules, things appear to be okay. Should I instead update the README to reflect the current state of the package?

Reply:
yes, update the README please

Reply:
actually, seeing what the readme says now, I would remove that, and add an [INSTALL.msg](https://github.com/void-linux/void-packages/blob/master/Manual.md#install_remove_files_msg) that says something like:

> Previous versions of solaar required copying or symlinking udev rules to /etc/udev/rules.d.
> If this was done previously, it is recommended to remove that file or symlink.

Reply:
Added the verbiage almost verbatim (attempted to honor the 80 character limit mentioned in the link). I also squashed the commits to remove the obsolete ones.